desktop/components/has_seen/dismisser.coffee

```diff
@@ -1,20 +1,24 @@
 _ = require 'underscore'
 Cookies = require '../cookies/index.coffee'
 
+# Get around inabilty to stub `document`
+domain = document.domain if typeof document isnt 'undefined'
+
 module.exports = class Dismisser
   defaults:
     limit: 5
     expires: 31536000 # 1 Year
+    domain: domain
```

Comment:
Sure, per @joeyAghion: `Cookies should hopefully be scoped to subdomains. (Especially sensitive ones, but all ideally.) Otherwise they’re vulnerable to subdomain takeovers from when we point DNS to services like Heroku, Marketo, Sailthru, etc.` (https://artsy.slack.com/archives/auctions-dev/p1488836444535829). 

There appeared to be some leakage between staging and prod yesterday which pointed to the fact that our cookies (for dismissing notifications) were not scoped to domain. Digging into the code, I noticed that this was in fact true.